integration/tests/cook/test_basic.py

```diff
@@ -650,8 +650,7 @@ def test_multiple_progress_updates_submit(self):
         command = f'{line_1} && sleep 2 && {line_2} && sleep 2 && ' \
                   f'{line_3} && sleep 2 && {line_4} && sleep 2 && ' \
                   f'{line_5} && sleep 2 && echo "Done" && exit 0'
-        job_uuid, resp = util.submit_job(self.cook_url, command=command, executor=job_executor_type,
-                                         max_runtime=60000, max_retries=5)
+        job_uuid, resp = util.submit_job(self.cook_url, command=command, executor=job_executor_type, max_retries=5)
```

Comment:
This change is unrelated to the rest of the PR, but the test flaked on this branch, so I'm fixing it here.